pydantic/config.py

```diff
@@ -555,15 +555,49 @@ class Transaction(BaseModel):
     3. Using `'never'` we would have gotten `user=SubUser(hobbies=['scuba diving'], sins=['lying'])`.
     """
 
+    val_date_or_time_unit: Literal['seconds', 'milliseconds', 'infer']
+    """The unit to assume for validating date or time strings. Defaults to `'infer'`.
+    Only applies when the date / time like input is an int / float.
+
+    The "epoch" reference below refers to the Unix epoch, which is 1970-01-01 00:00:00 UTC.
+
+    - `'seconds'` will validate date or time strings as seconds since the epoch.
+    - `'milliseconds'` will validate date or time strings as milliseconds since the epoch.
+    - `'infer'` will infer the unit from the string based on unix time:
```

Comment:
numeric not stringy inputs? also "since the epoch" is specific to datetime, timedelta has no such anchor

```suggestion
    - `'seconds'` will validate date or time numeric inputs as seconds.
    - `'milliseconds'` will validate date or time numeric inputs as milliseconds .
    - `'infer'` will infer the unit from the numeric input based on unix time:
```